examples/lighting-app/nrfconnect/README.md

```diff
@@ -210,13 +210,18 @@ To use the Docker container for setup, complete the following steps:
     -   _-e RUNAS=\$(id -u)_ is needed to start the container session as the
         current user instead of root.
 
-4.  Check out or update the nRF Connect SDK to the recommended `v1.5.0` version
-    by running the following command in the Docker container:
+4.  Check out or update the nRF Connect SDK to the recommended version.
+    
+    If you would like to check, if the nRF Connect SDK version you currently have is the recommended one, run the following commands in the Docker container. Replace _chip-dir_ with the path to the main CHIP repository directory.
 
-         $ setup --ncs v1.5.0
-         /var/ncs repository is empty. Do you wish to check out nRF Connect SDK sources [v1.5.0]? [Y/N] y
-         ...
-         /var/chip repository is initialized, skipping...
+         $ cd chip-dir
+         $ python3 scripts/setup/nrfconnect/update_ncs.py --check
```

Comment:
There should be no harm in running `--update`, so we could simplify the instruction a bit. Also, in Docker `chip-dir` == `/var/chip`, so another simplication can be applied.